src/launchpad/size/analyzers/apple.py

```diff
@@ -259,6 +259,23 @@ def analyze(self, artifact: AppleArtifact) -> AppleAnalysisResults:
 
         return results
 
+    def parse_plist_date(self, date_value: str | datetime | None) -> str | None:
+        if date_value is None:
+            return None
+
+        if isinstance(date_value, str):
+            try:
+                dt = datetime.fromisoformat(date_value)
+                return dt.isoformat()
+            except (ValueError, AttributeError):
+                # If parsing fails, return the original string
+                # This is defensive - we don't want to break on unexpected formats
+                logger.debug(f"Could not parse date string: {date_value}")
+                return date_value
+
+        # isinstance(date_value, datetime)
```

Comment:
Accidental comment?

Reply:
no intentional, its part of the type flow